src/components/emoji-picker/EmojiPicker.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { EmojiPicker } from './EmojiPicker';
+
+export default {
+  title: `Components/EmojiPicker`,
+  component: EmojiPickerDemo,
+  argTypes: {
+  },
+};
+
+function EmojiPickerDemo() {
```

Comment:
@steiningermona  like this you can not see the code on "Docs" please check the story of SSelectSearchOperatorDropdown.stories 
you need to remove the wrapping function